.github/workflows/docs.yml

```
@@ -0,0 +1,83 @@
+        run: uv run python scripts/generate_markdown_inventory.py --output docs/MARKDOWN_INVENTORY.md
+
+      - name: Preserve rendered inventory
+        uses: actions/upload-artifact@v7
+        with:
+          name: markdown-inventory
+          path: docs/MARKDOWN_INVENTORY.md
+          if-no-files-found: error
+
+      - name: Sync generated inventory on factory pull requests
+        if: github.event_name == 'pull_request' && startsWith(github.event.pull_request.head.ref, 'factory/')
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)